test/unit/lookup_key_test.rb

```diff
@@ -24,7 +24,7 @@ def test_path2match_single_domain_path
     key   = ""
     value = ""
     as_admin do
-      key   = LookupKey.create!(:key => "ntp", :path => "domain", :puppetclass => Puppetclass.first)
+      key   = LookupKey.create!(:key => "ntp", :path => "domain", :puppetclass => Puppetclass.first, :key_type => "string")
```

Comment:
This might hint at a bug - both key_type and validator_type may be nil or blank, but it looks like a TypeError would be raised now.  It might have been handled by the respond_to? in cast_validate_value before.